code/build.R

```diff
@@ -224,7 +224,7 @@ if (GENERATE_CDB) {
       cycle_net_traffic = left_join(cycle_net_joined, cycleways_with_traffic_df)
 
       cycle_net_traffic_na = cycle_net_traffic |>
-        filter(highway %in% c("residential", "tertiary", "service"))
+        filter(highway %in% c("residential", "tertiary", "service", "primary", "tertiary_link", "secondary", "trunk","primary_link","living_street", "unclassified"))
```

Comment:
What do these additional bits do?

Reply:
You could use 

```suggestion
        filter(str_detect(highway, "residential|tertiary|service|primary|secondary|trunk|living"))
```
As a more general solution.

Reply:
And it's more concise.

Reply:
I don't think we need to estimate traffic volumes for "unclassified" roads, right?

Reply:
agree

Reply:
Relevant bit of code from osmactive: https://github.com/nptscot/osmactive/blob/8e86f7f138486f55e69402583bb841d7de4fbc55/R/osmactive.R#L807-L823